better-font-awesome.php

```diff
@@ -142,7 +147,7 @@ public static function get_instance( $args = '' ) {
      *
      * @since  0.9.0
      */
-    function __construct() {
+    private function __construct() {
```

Comment:
I think you can serve the same purpose here with ```protected``` and allow more flexibility with extending classes